keyboards/bear_face/rules.mk

```
@@ -29,3 +21,5 @@ MIDI_ENABLE = no            # MIDI controls
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
```

```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output6
```
And could you remove the `MIDI_ENABLE` line above this as well? (GitHub doesn't let me extend the suggestion).